dev-requirements.txt

```diff
@@ -1,9 +1,10 @@
-git+https://github.com/jupyterhub/the-littlest-jupyterhub
-git+https://github.com/plasmabio/tljh-repo2docker
-jupyterhub~=1.5
-notebook<7
+git+https://github.com/jupyterhub/the-littlest-jupyterhub@1.0.0
+git+https://github.com/jtpio/tljh-repo2docker@jupyterhub-4
```

Comment:
This can be updated when there is a proper tag on https://github.com/plasmabio/tljh-repo2docker that includes the changes from https://github.com/plasmabio/tljh-repo2docker/pull/65.